content/docs/04-clusters/03-edge/09-site-deployment/05-site-installation/06-edge-host-registration.md

```diff
@@ -34,9 +40,11 @@ Select the registration method that best fits your organizational needs and revi
 
 ## Auto Registration
 
-You can automate the registration process by using registration tokens. This method requires providing the registration token in the user data. If a default project was selected for the registration token, that is the project the Edge host will be registered under. You can override the default project by providing a project ID in the user data.
+You can automate the registration process by using registration tokens. This method requires providing the registration token in the user data. To complete the auto-registration process, it is mandatory to assign a project when creating a registration token. 
```

Comment:
```suggestion
You can automate the registration process by using registration tokens.  
```
We can remove it here because it's mandatory and stated in the intro. The last sentence is the Slack thread I mentioned you a few days ago where they changed the behavior.